base/REPLCompletions.jl

```diff
@@ -222,14 +222,61 @@ get_value(sym::Symbol, fn) = isdefined(fn, sym) ? (fn.(sym), true) : (nothing, f
 get_value(sym::QuoteNode, fn) = isdefined(fn, sym.value) ? (fn.(sym.value), true) : (nothing, false)
 get_value(sym, fn) = sym, true
 
+# Return the value of a getfield call expression
+function get_value_getfield(ex::Expr, fn)
+    val, found = get_value_getfield(ex.args[2],fn)
```

Comment:
can we help the reader with a comment about what is contained in `ex.args[2]` and `ex.args[3]`?